checkov/terraform/checks/resource/gcp/GoogleComputeFirewallUnrestrictedIngress21.py

```diff
@@ -0,0 +1,16 @@
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.gcp.AbsGoogleComputeFirewallUnrestrictedIngress import AbsGoogleComputeFirewallUnrestrictedIngress
+
+PORT = 21
```

Comment:
I'm thinking, if we also should add a check for the FTP data port 20 🤔 wdyt?

Reply:
Its not like its tough challenge! What about the rest of these while were at it? https://www.trendmicro.com/cloudoneconformity/knowledge-base/gcp/CloudVPC/

Reply:
![](https://media.giphy.com/media/kQ3FSVoJrkYWk/giphy.gif)

Reply:
how about we recommend to turn off ingress completely 😄